proxy/image.yaml

```diff
@@ -32,10 +32,10 @@ packages:
   content_sets:
     x86_64:
       # Required for tini
-      - amq-streams-2-for-rhel-8-rpms
+      - amq-streams-2-for-rhel-9-rpms
       # Required for base image
-      - rhel-8-for-x86_64-baseos-eus-rpms__8_DOT_6
-      - rhel-8-for-x86_64-appstream-eus-rpms__8_DOT_6
+      - rhel-9-for-x86_64-baseos-eus-rpms__9_DOT_2
+      - rhel-9-for-x86_64-appstream-eus-rpms__9_DOT_2
```

Comment:
The line below "nss-3.67.0-7.el8_5 # experiment - try pinning to the FIPS validated nss." needs to be removed.  This related to RHEL 8.6.

Reply:
@k-wall @MikeEdgar I've double checked and those packages are not present in the RHE-9 image.  I'll remove above packages from removal list.

Reply:
Is there anything coming from RHEL 9 base image that is superfluous to us?  If so, we should remove it/them